dotCMS/src/main/java/com/dotcms/graphql/business/CountTypeProvider.java

```diff
@@ -0,0 +1,28 @@
+package com.dotcms.graphql.business;
+
+import static com.dotcms.util.CollectionsUtils.map;
+import static graphql.Scalars.GraphQLString;
+
+import com.dotcms.graphql.util.TypeUtil;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLOutputType;
+import graphql.schema.GraphQLType;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+
+public enum CountTypeProvider implements GraphQLTypesProvider {
+
+    INSTANCE;
+
+    final Map<String, GraphQLOutputType> pageFields = map(
+            "totalCount", GraphQLString,
```

Comment:
should it be the same name than here https://github.com/dotCMS/core/pull/19032/files#diff-cf87643edb0d8a8ccf07993e94b7180bR21?

maybe we can create a suoer class here to define to add the count field and create the Map, and later each sub class could add the extra parameters

Reply:
Waiting on discussion with team

Reply:
leaving it like this